.drone.yml

```diff
@@ -1,80 +1,132 @@
 ---
 kind: pipeline
 name: linux-amd64
+
 platform:
   os: linux
   arch: amd64
 
 steps:
-- name: ci
-  image: briandowns/rancher-build-base:v0.1.1
+- name: build
+  image: rancher/dapper:v0.5.4
   commands:
-  - ./scripts/ci
+    - dapper ci
+  volumes:
+    - name: docker
+      path: /var/run/docker.sock
   when:
     event:
     - push
     - pull_request
     - tag
 
-- name: build
-  image: briandowns/rancher-build-base:v0.1.1
-  volumes:
-  - name: dockersock
-    path: /var/run
-  commands:
-  - sleep 20
-  - TAG=${DRONE_TAG} make
+- name: docker-publish
+  image: plugins/docker
+  settings:
+    dockerfile: package/Dockerfile
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+    repo: "rancher/eks-operator"
+    tag: "${DRONE_TAG}-amd64"
```

Comment:
@cbron with these tags, is the expectation that the user will retag it without the arch so it's compatible with the charts or do we need to add some logic to the charts?

Reply:
If you tag v1.0.0 this would create v1.0.0-amd64 and v1.0.0-arm64, but manifest will unify them into just 1.0.0. User doesn't tag anything around arch's.